sentinel/transitions/index.js

```diff
@@ -324,15 +305,31 @@ const finalize = ({ change, results }, callback) => {
 const applyTransition = ({ key, change, transition }, callback) => {
 
   const _setResult = ok => {
-    const doc = change.doc;
-    if (!doc.transitions) {
-      doc.transitions = {};
-    }
-    doc.transitions[key] = {
-      last_rev: parseInt(change.doc._rev) + 1,
-      seq: change.seq,
-      ok: ok
-    };
+    return new Promise((resolve, reject) => {
+      const info = change.info;
+      info._id = info._id || `${change.id}-info`;
+
+      dbPouch.sentinel.get(info._id)
+      .then(doc => { return doc; })
+      .catch(err => {
+        if(err.status === 404) { return info; }
```

Comment:
Although it doesn't explicitly say it, our [style guide implies the `return info` goes on a new line](https://github.com/medic/medic-docs/blob/master/development/style-guide.md#opening-braces-go-on-the-same-line), eg:

```js
if (err.status === 404) {
  return info;
}
```

Try to keep your changes in keeping with the general style of the rest of the code.